test/functional/signrawtransactions.py

```diff
@@ -85,11 +95,37 @@ def script_verification_error_test(self):
         # Make sure decoderawtransaction throws if there is extra data
         assert_raises(JSONRPCException, self.nodes[0].decoderawtransaction, rawTx + "00")
 
+        rawTxSigned = self.nodes[0].signrawtransactionwithkey(rawTx, privKeys, scripts)
+
+        # 3) The transaction has no complete set of signatures
+        assert 'complete' in rawTxSigned
+        assert not rawTxSigned['complete']
+
+        # 4) Two script verification errors occurred
+        assert 'errors' in rawTxSigned
+        assert_equal(len(rawTxSigned['errors']), 2)
+
+        # 5) Script verification errors have certain properties
+        assert 'txid' in rawTxSigned['errors'][0]
```

Comment:
As above, you don't need to test presence of these keys if you're going to test the values below. Recommend you flip `# 5` and `# 6` sections, and only test presence of keys where you haven't tested the values.

Reply:
These were copied from a previously existing test.

Reply:
ok, I don't think there's any need to copy bad patterns, but I'm not too concerned either way.

Reply:
Agree with @jnewbery.